README.md

```diff
@@ -1,2 +1,54 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Portafolio
-This project shows all my experience as a software Developer
+It is my portfolio, experience has a software developer
+
+> Description the project.
+- **This portfolio show my experience as a software developer.**
+> ## Objectives
+- Build a Mobile First portafolio
+- Follow the best practices of adabtative content and Mobile First
+- Setup an skeleton 
+- Use flexBox
+- Follow GitHub flow.
+- Open pull requests in GitHub.
+
+
+## Built With
+- HTML
+- CSS
+
+## Live Demo (if available)
+we don't have a demo yet
+
+
+## Getting Started
+
+
+
+
+## Authors
+
+👤 **Luis David Rodriguez Valades**
+
+- GitHub: [@githubhandle](https://github.com/LuisDavidRodriguez)
+- LinkedIn: [profile](https://www.linkedin.com/in/luis-david-rodriguez-valades-24a0a8239)
+- Youtube: [chanel](https://www.youtube.com/channel/UChuA4SgdDYk2DHStsy7HEgQ)
+
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+
+## 📝 License
```

Comment:
- [ ]  [OPTIONAL] it would be more useful  to include  a license to your project.  otherwise Kindly consider removing  this sections from your readmebecause it  isn't being used.